packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.0
```

Comment:
Is this worth a minor bump, or should I make it a patch 2.0.8?

Reply:
I would say 2.0.8; it seems much more like a bugfix than a feature.

Reply:
Done.